### ISSUE TITLE
Changing language does not work

### ISSUE THREAD

**Issue by jonashaag** [victorjonsson/jQuery-Form-Validator#442]
I think it's due to the refactoring done in February. Internally `.language` is used in some places but it's never assigned to anywhere:

``` sh
$ ag '\.language' src/
src/main/deprecated.js
40:            config.language.errorTitle,

src/main/setup.js
114:        var valid = $form.isValid(conf.language, conf);
145:        $form.validateOnBlur(conf.language, conf);
147:          $form.validateOnBlur(conf.language, conf);
151:        $form.validateOnEvent(conf.language, conf);

src/modules/toggleDisabled.js
56:              if( valid && $form.isValid(conf.language, conf, false) ) {
73:      $formsToDisable.validateOnEvent(conf.language, conf);
```

``` sh
$ ag '\blanguage\b.+=' src/
src/main/jquery-plugins.js
138:    var language = $.extend({}, $.formUtils.LANG, lang || {});
202:    language = $.extend({}, $.formUtils.LANG, language || {});
276:    language = $.extend({}, $.formUtils.LANG, language || {});

src/main/utils.js
132:      language = language || $.formUtils.LANG;
```

**Comment by victorjonsson** [comment]
I'm unable to reproduce this bug http://jsbin.com/rowiyegako/edit?html,output?

Could you please provide me with your implementation code?

**Comment by jonashaag** [comment]
Hm, ok, it seems to be a problem with webpack/module loading. I'm using webpack to combine/compress JS, using these imports:

```
import $ from 'jquery';
require('jquery-form-validator/form-validator/jquery.form-validator.js');
require('jquery-form-validator/form-validator/lang/de.js');
```

Things is that jquery form validator tries to load `de.js` from an external source, which fails. My workaround is to simply inline the language definition, but maybe there is a better way?

**Comment by victorjonsson** [comment]
Yes, it seems that the localization mechanism doesn't support this case. I will have it fixed soon.

**Comment by jonashaag** [comment]
Awesome! Thanks a lot!

**Comment by Perde1989** [comment]
Hello, what is the current state of this issue? I am facing the same problem (I use RequireJS).

**Comment by jonashaag** [comment]
@Perde1989 we just worked around this issue by inlining the language definition in our Javascript code.

**Comment by cytec** [comment]
any news on this? 
If i minify scripts language files don't load like they should and defining a custom lanugage object like shown here: http://www.formvalidator.net/#localization doesn't work and console says: `Unable to load form validation module ..../Object].js`

**Comment by victorjonsson** [comment]
This should now have been fixed as of version 2.3.61
